javascript-data-model/order-history.js

```diff
@@ -0,0 +1,73 @@
+var orderHistory = [
+  {
+    placed: 'August 4, 2020',
+    total: '$34.00',
+    shipTo: 'JS Masher',
+    orderNum: 'ORDER # 114-3941689-8772232',
+    delivered: 'Delivered Aug 8, 2020',
+    items: [
+      {
+        item: 'JavaScript for impatient programmers',
+        img: '',
+        author: 'Rauschmayer, Dr. Axel',
+        price: '$31.55',
+        returnWindow: 'Return window closed on Sep 7, 2020'
+      }
+    ]
+  },
+  {
+    placed: 'July 19, 2020',
+    total: '$44.53',
+    shipTo: 'JS Masher',
+    orderNum: 'ORDER # 113-9984268-1280257',
+    delivered: 'Delivered Jul 20, 2020',
+    packageLeft: 'Your package was delivered. It was handed directly to a resident',
+    items: [
+      {
+        item: 'The Timeles Way of Building',
+        img: '',
+        author: 'Alexander, Chrisopher',
+        price: '$41.33',
+        returnWindow: 'Return window closed on Aug 19, 2020'
+      }
+    ]
+  },
+  {
+    placed: 'July 4, 2020',
+    total: '$17.22',
+    shipTo: 'JS Masher',
+    orderNum: 'ORDER # 114-2875557-9059409',
+    delivered: 'Delivered Jul 7, 2020',
+    packageLeft: 'Your package was delivered. It was handed directly to a resident.',
```

Comment:
This `packageLeft` property is only on some of the orders, but it seems like it would be relevant to all orders. If there was no special message, I would still have the property, but set it to an empty string or `null`